content/en/integrations/guide/source-code-integration.md

```diff
@@ -226,7 +236,9 @@ You can see links from stack frames to their source repository in [Error Trackin
 2. Click on an issue. The **Issue Details** panel appears on the right.
 3. Under **Latest Event**, click the **View** button on the right of a frame or select **View file**, **View Git blame**, or **View commit** to be redirected to your source code management tool.
 
-{{< img src="integrations/guide/source_code_integration/links-to-git-error-full.png" alt="A view repository button with three options (view file, view blame, and view commit) available on the right side of an error stack trace in Error Tracking" style="width:100%;">}}
+{{< img src="integrations/guide/source_code_integration/error-tracking-panel-full.png" alt="A view repository button with three options (view file, view blame, and view commit) available on the right side of an error stack trace in Error Tracking, along with inline code snippets in the stack trace" style="width:100%;">}}
```

Comment:
Can you confirm that this screenshot was taken on Demo or a private sandbox account? I wasn't able to find this issue in Demo, so just want to double-check.